docs/steps.rst

```diff
@@ -28,3 +61,107 @@ Tools for step writing
 .. automodule:: aloe.tools
     :members:
 
+Writing good BDD steps
+----------------------
+
+It's very easy with BDD testing to accidentally reinvent Python testing
+using a pseudo-language. Doing so removes much of the point of using BDD
+testing in the first place, so here is some advice to help write better BDD
+steps.
+
+1.  **Be declarative not imperative**
```

Comment:
Worthy goal, but this is not what's described here. The BDD steps are imperative by nature, compare: "I log on as danni" and "When I am logged in as danni".

What I think you mean is just not going too deep into the implementation details, perhaps the title should be "Avoid implementation details".